awx/main/validators.py

```diff
@@ -76,7 +76,7 @@ def validate_pem(data, min_keys=0, max_keys=None, min_certs=0, max_certs=None):
         if pem_obj_type.endswith('PRIVATE KEY'):
```

Comment:
According to:

https://access.redhat.com/solutions/1519083

> The ssh-keygen command--as shipped in every modern version of RHEL--generates keys in a format (RFC4716) that necessitates using the MD5 digest algorithm at some point in the process (as part of a transformation performed on the input passphrase)

> On a FIPS-enforcing system:

> MD5 is completely disabled so virtually all encrypted SSH keys (private keys protected by a passphrase) will not be usable

> OpenSSL (called by ssh-keygen) automatically generates PKCS8-formatted keys without MD5 (when run on a system in FIPS mode)

In order to accommodate this, we need to change our input validation _slightly_ so that it can detect keys in PKCS8 format (because users that enable FIPS will have to specify their keys in this format for passphrase to work).